FortniteApi/Services/FortniteService.cs

```diff
@@ -30,9 +50,13 @@ public async Task<CosmeticResponseDto> CreateCosmetic(CreateCosmeticDto cosmetic
         return cosmetic.ToResponseDto();
     }
 
-     private async Task<bool> CosmeticAlreadyExist(string name, CancellationToken cancellationToken)
+    private async Task<bool> CosmeticAlreadyExist(string name, CancellationToken cancellationToken)
     {
         var cosmetic = await _fortniteRepository.GetByNameAsync(name, cancellationToken);
         return cosmetic != null;
```

Comment:
Aqui puedes reutilizar el CosmeticExists, algo como:

```suggestion
        return CosmeticExist(cosmetic);
```